_includes/pbc/all/install-features/202304.0/install-the-inventory-management-feature.md

```diff
@@ -0,0 +1,725 @@
+    {
+        return [
+            new StockAddressStockPostCreatePlugin(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Zed\StockExtension\Dependency\Plugin\StockPostUpdatePluginInterface[]
+     */
+    protected function getStockPostUpdatePlugins(): array
+    {
+        return [
+            new StockAddressStockPostUpdatePlugin(),
+        ];
+    }
+}
+```
+</details>
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the warehouse address management works:
+1. In the Back Office, create a warehouse.
+2. Import a warehouse address using a data import functionality.
+3. Check if the imported warehouse address exists in the `spy_stock_address` database table.
+
+{% endinfo_block %}
+
+4. Register the following plugins for warehouse allocation:
+
+| PLUGIN                                          | SPECIFICATION                             | PREREQUISITES | NAMESPACE                                                |
+|-------------------------------------------------|-------------------------------------------|---------------|----------------------------------------------------------|
+| SalesOrderWarehouseAllocationCommandPlugin      | Allocates warehouse for sales order item. | none          | Spryker\Zed\WarehouseAllocation\Communication\Plugin\Oms |
+
+
+**src/Pyz/Zed/Oms/OmsDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Oms;
+
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\Oms\Dependency\Plugin\Command\CommandCollectionInterface;
+use Spryker\Zed\Oms\OmsDependencyProvider as SprykerOmsDependencyProvider;
+use Spryker\Zed\WarehouseAllocation\Communication\Plugin\Oms\SalesOrderWarehouseAllocationCommandPlugin;
+
+class OmsDependencyProvider extends SprykerOmsDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function extendCommandPlugins(Container $container): Container
+    {
+       $container->extend(self::COMMAND_PLUGINS, function (CommandCollectionInterface $commandCollection) {
+            $commandCollection->add(new SalesOrderWarehouseAllocationCommandPlugin(), 'WarehouseAllocation/WarehouseAllocate');
+
+            return $commandCollection;
+        });
+
+        return $container;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that after the order is created, order items gain the `warehouse allocated` status.
+
+{% endinfo_block %}
+
+
+## Install an example of product and product offer warehouse allocation
+
+Follow the steps below to install an example for product and product offer warehouse allocations.
+
+### Prerequisites
+
+To start integration, integrate the required features:
+
+| NAME         | VERSION          | INTEGRATION GUIDE |
+|--------------|------------------|------------------|
+| Inventory Management | {{site.version}} |[Inventory Mamagement feature integration](#install-feature-core) |
```

Comment:
@sakharova-yuliia I added a link to the beginning of the document because prerequisites say we must integrate the feature itself first.  Correct me if I'm wrong.

Reply:
Yes, the feature itself has to be integrated before example modules.